src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -15,7 +17,7 @@
 
     private OperatorInterface oi;
 
-    private SupplyCurrentLimitConfiguration climberLimit = new SupplyCurrentLimitConfiguration(true, 20, 20, 0);
+    private SupplyCurrentLimitConfiguration climberLimit = new SupplyCurrentLimitConfiguration(true, 40, 40, 0);
```

Comment:
This seems too conservative for the climber. 2019 climber used `new SupplyCurrentLimitConfiguration(true, 120, 90, 300);`

Reply:
should be good now